### PR TITLE
Crypto

### PR DESCRIPTION
**Pull request by JGarcia1998**

### CODE REVIEW

File: stock-server/app.js [hunk 1]
```diff
@@ -40,6 +40,68 @@ app.post("/create-account", (req, res) => {
   res.send({ message: "Account created successfully" });
 });
 
+app.get("/user-watchlist/:id", (req, res) => {
```
Comment (mikelleu) [inline stock-server/app.js:43]: this app's backend is currently fairly small. but its good practice to abstract away your api endpoints, db operations, and server initialization. 

so having a file with just endpoints. just db operations etc.

**Review by mikelleu — CHANGES_REQUESTED** [review]

**Comment by mikelleu** [comment]
using a linter (eslint) and a formatter (prettier) will help you with a lot of these issues.
use them and pay attention to the linter and their suggestions

File: stock-client/src/App.css [hunk 2]
```diff
@@ -285,7 +285,7 @@ body {
     width: 1rem;
     height: 1rem;
     opacity: 0;
-    background-color: #3e80fa;
```
[the remark this answers is not in the document]
Comment (JGarcia1998) [reply stock-client/src/App.css:288]: I use prettier already, this is the compiled css file from my sass compiler. I just run the compile script after writing scss and it compiles to this one css file thats why it looks ugly

File: stock-client/src/components/Container.js [hunk 3]
```diff
@@ -6,8 +6,12 @@ import { useEffect, useState } from "react";
...
 import Button from "@material-ui/core/Button";
+import Snackbar from "@material-ui/core/Snackbar";
+import IconButton from "@material-ui/core/IconButton";
+import CloseIcon from "@material-ui/icons/Close";
 
 function Container(props) {
+  const [open, setOpen] = React.useState({ open: false, message: "" });
   const useStyles = makeStyles((theme) => ({
     root: {
       "& > *": {
```
[the remark this answers is not in the document]
Comment (JGarcia1998) [reply stock-client/src/components/Container.js:17]: this was code from material ui, ill try removing it and see if It changes much

File: stock-client/src/components/Container.js [hunk 4]
```diff
@@ -16,20 +20,74 @@ function Container(props) {
...
+  const handleClose = (event, reason) => {
+    if (reason === "clickaway") {
+      return;
+    }
+    setOpen({ open: false, message: "" });
+  };
+
+  useEffect(() => {
+    fetch(
+      `https://newsapi.org/v2/top-headlines?country=us&apiKey=${process.env.REACT_APP_KEY}`
```
[the remark this answers is not in the document]
Comment (JGarcia1998) [reply stock-client/src/components/Container.js:33]: How else can I hide my api keys in client side? This was the solution I found, but I did see something that said it is still possible to get the api key if someone wanted to

File: stock-client/src/App.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (mikelleu) [reply stock-client/src/App.css:288]: Gotcha. That's why is so huge

File: stock-client/src/components/Container.js [hunk 5]
```diff
@@ -16,20 +20,74 @@ function Container(props) {
...
+          anchorOrigin={{
+            vertical: "bottom",
+            horizontal: "center",
+          }}
+          open={open.open}
+          autoHideDuration={6000}
+          onClose={handleClose}
+          message={open.message}
+          action={
+            <React.Fragment>
```
[the remark this answers is not in the document]
Comment (JGarcia1998) [reply stock-client/src/components/Container.js:78]: This is also from material ui, I didn't realize these meant the same thing

**Pull request merged**